src/app/[locale]/globals.css

```diff
@@ -7,7 +7,89 @@
 		text-wrap: balance;
 	}
 
+	/* 
+	===============
+	Extra Heading 
+	===============
+	*/
 	.vm-text-eh1 {
 		@apply text-6xl font-semibold leading-[70px] tracking-tight;
 	}
+	.vm-text-eh2 {
+		@apply text-4xl font-semibold leading-[53.2px] tracking-tight;
+	}
+	.vm-text-eh3 {
+		@apply text-3xl font-semibold leading-[44.8px] tracking-tight;
+	}
+	.vm-text-eh4 {
+		@apply text-[28px] font-semibold leading-[39.2px] tracking-tight;
+	}
+	.vm-text-eh5 {
+		@apply text-2xl font-semibold leading-[33.6px] tracking-tight;
+	}
+	.vm-text-eh6 {
+		@apply text-2xl font-semibold leading-[33.6px] tracking-tight;
+	}
+
+	/* 
+	===============
+	Heading
+	===============
+	*/
+	.vm-text-h1 {
+		@apply text-[34px] font-semibold leading-[40.8px]  tracking-tight;
+	}
+	.vm-text-h2 {
+		@apply text-lg font-semibold leading-7 tracking-tight;
+	}
+	.vm-text-h3 {
+		@apply text-base font-semibold leading-5 tracking-tight;
+	}
+	.vm-text-h4 {
+		@apply text-sm font-semibold leading-4 tracking-tight;
+	}
+	.vm-text-h5 {
+		@apply text-xs font-semibold leading-4 tracking-tight;
+	}
+	.vm-text-h6 {
+		@apply text-[10px] font-semibold leading-4 tracking-tight;
+	}
+
+
+
+	/* 
+	===============
+	Paragraphs
+	===============
+	*/
+	.vm-text-p1-regular{
+		@apply text-lg font-normal leading-7
+	}
+	.vm-text-p1-medium{
+		@apply text-lg font-medium leading-7
+	}
+	.vm-text-p2-regular{
+		@apply text-base font-normal leading-7
+	}
+	.vm-text-p2-medium{
+		@apply text-base font-medium leading-7
+	}
+	.vm-text-p3-regular{
+		@apply text-sm font-normal leading-7
+	}
+	.vm-text-p3-medium{
+		@apply text-sm font-medium leading-7
+	}
+	.vm-text-p4-regular{
+		@apply text-xs font-normal leading-7
+	}
+	.vm-text-p4-medium{
+		@apply text-xs font-medium leading-7
+	}
+	.vm-text-p5-regular{
+		@apply text-[10px] font-normal leading-7
+	}
+	.vm-text-p5-medium{
+		@apply text-[10px] font-medium leading-7
+	}
```

Comment:
Do we have these in the design? If not, please remove.

Reply:
We do.
it is in the paragraph section of the typography component

Reply:
Oh I see, the file had two P1 stylings. Ok we can resolve this one

Reply:
Yeah it does.
One for regular font-weight, the other for medium font-weight